services/alerts/src/types.ts

```diff
@@ -0,0 +1,31 @@
+type AlertAction = {
+    label: string
+    onClick: Function
+}
+export type AlertOptions = {
+    actions?: AlertAction[]
+    className?: string
+    critical?: boolean
+    dataTest?: string
+    duration?: number
+    icon?: boolean | React.ElementType
+    permanent?: boolean
+    success?: boolean
+    warning?: boolean
+    onHidden?: Function
```

Comment:
The reason I suggested this is: I don't like pinning types on functions that come from the outside where we don't do anything with the return value and we don't pass any arguments to. It's basically irrelevant what that function does and if the app wants to use that function somewhere else, it wouldn't have to create a separate one because of a type declaration that doesn't need to be there.

This only concerns functions that we don't pass any arguments to and do not use the return value

Reply:
I discussed this with Austin on Slack and he provided [an example](https://www.typescriptlang.org/play?#code/C4TwDgpgBA8mwEsD2A7AzlAvFA3gKCkKlQAkEATciFALigAoBKLAPigDckK8BfPPAPQCAKgAsEGYKIBOSAO4YAhiigRps6XgDGqNMGLxk6AILSA5nTiJdWXASKkKVWg0U1lIZpjaKoAaigAFgAmXn4hMQkociQINBQAcn0pWTkoZVV1JGkoACMILUUAVzRoKWhFcyKAW2p9KKRDVEUAG21dfUbrdCsjVtMLWCb0W3wiYhQySmo6ekUAfncUT1Z0-yDQvkEBKEiMOWyAazR29E7htAAlCGAi6Rdem2wxh0mnGYYvNhCw7d3xfZHNAAGigaAQKC00AQwASGHYrQoUGASCgOnQSBaEAAdC0kGZ0sspBCzKc9FAAB50ThI7BdIxXG53FDYxzTFBMMmYnF4sz0CmMcI7PZQABmigQLRBeQKxVKHC45CgURQSH0ijQaAlKEUuSxyNRvhQNXyOXo2UJIGI5Ry0ggrWR4AggvR5JAdGN1VNtgp4QAVP9oHaNahlZJRIp1WKipDup9VjTyKDylA7bd7hxWkVoAB1ACSABkC1AAHIAUQAamXLjKoCUIEnlfo5JKWsiZPI1qBIJkNNioABlVEwuFgxSisqotN3ahrBEtbOg8GQ6HN1tQFAQdhqdKiidadVE8QoAnKJUAKxK+ny7dSKkUckUVr9AiAA) for why `() => void` is suitable here.